Solutions/Windows Security Events/Hunting Queries/Suspicious_Windows_Login_outside_normal_hours.yaml

```diff
@@ -1,6 +1,8 @@
 id: e7bfbc3f-98c7-4aaa-a64c-de9c058b86b2
 name: Suspicious Windows Login Outside Normal Hours
 description: |
+  'Query identifies unusual logon events outside a user's normal hours by comparing with the last 14 days, flagging anomalies based on historical patterns.'
```

Comment:
Query identifies unusual logon events outside a user's normal hours by comparing with the last 14 days' logon activity, flagging anomalies based on historical patterns.